LoRaEngine/test/LoRaWan.IntegrationTest/C2DMessageTest.cs

```diff
@@ -35,7 +35,7 @@ public C2DMessageTest(IntegrationTestFixtureCi testFixture)
         /// <summary>
         /// Ensures that a cloud to device message has not been seen more than expected
         /// </summary>
-        /// <param name="foundCount"></param>
+        /// <param name="foundCount">foundcount.</param>
```

Comment:
Why is there a difference in casing between foundCount and foundcount. ? If it's descriptive, I think "foundcount" doesn't really explain that much.

Reply:
It triggered a checkstyle error, that is why. I think the variable name explain well enough what it does. I will replace with "Count of found item", i hope it will help more :)